install.sh

```diff
@@ -77,27 +112,49 @@ v sudo usermod -aG video "$(whoami)"
 v sudo usermod -aG input "$(whoami)"
 
 #####################################################################################
-printf '\e[36m2. Installing AGS from git repo\e[97m\n'
+printf '\e[36m2. Installing AGS and rubik from git repo\e[97m\n'
 sleep 1
 
-installags (){
-  v git clone --recursive https://github.com/Aylur/ags.git|| \
-    if [ -d ags ];then
-      printf "\e[36mSeems \"./ags\" already exists.\e[97m\n"
-      cd ags
-      v git pull
-    else exit 1
-    fi
-  v npm install
-  v meson setup build 
-  v meson install -C build
+install-ags (){
+  mkdir -p $base/ags
+  cd $base/ags
+  try git init -b main
+  try git remote add origin https://github.com/Aylur/ags.git
+  git pull origin main && git submodule update --init --recursive
+  npm install
+  meson setup build 
+  meson install -C build
+  cd $base
+}
+install-rubik (){
```

Comment:
- If you're going to handle font installation, it'd be nice if you could make it install the font "Gabarito" too
  - It's used in title texts
  - I think it's here https://github.com/naipefoundry/gabarito
  - But it's up to you. I'll let you decide whether or not to add this, no pressure

Reply:
Done.